blocks/activity-list/index.js

```diff
@@ -1,83 +1,55 @@
-// jscs:disable disallowMultipleLineStrings
-// jscs:disable validateQuoteMarks
 /* eslint-disable space-in-parens */
 
 /**
  * Internal block libraries.
  */
+const { registerBlockType } = wp.blocks;
+const { InspectorControls } = wp.editor;
+const { ServerSideRender, TextControl, PanelBody } = wp.components;
+const { Fragment } = wp.element;
 const { __ } = wp.i18n;
-const { registerBlockType, InspectorControls } = wp.blocks;
-const { Placeholder, withAPIData, TextControl, PanelBody } = wp.components;
-const { RawHTML } = wp.element;
-const { decodeEntities } = wp.utils;
+
+const blockName = 'amp-travel/activity-list';
 
 /**
  * Register block.
  */
 export default registerBlockType(
-	'amp-travel/activity-list',
+	blockName,
 	{
-		title: __( 'Activity list' ),
+		title: __( 'Activity list', 'travel' ),
 		category: 'common',
 		icon: 'tickets',
 		keywords: [
-			__( 'Browse activities' ),
-			__( 'Travel' )
+			__( 'Browse activities', 'travel' ),
+			__( 'Travel', 'travel' )
 		],
 
-		edit: withAPIData( () => {
```

Comment:
`withAPIData` was deprecated (removed). There is `wp.apiFetch()`, but that returns a promise. I thought it'd be simpler to use the `<ServerSideRender>` component instead.